testdata/BUILD

```diff
@@ -104,6 +104,12 @@ container_image(
     tars = ["one.tar"],
 )
 
+container_image(
+    name = "tar_with_rpm",
+    base = ":tar_base",
+    rpms = ["iptables-1.6.2-3.fc27.i686.rpm"],
```

Comment:
IIUC I firstly will need to download `rpm` under the directory with the bazel target, does it correct?

Reply:
Yes. Correct. An example download rule would look like this:

```
http_file(
   name = "glibc",
   url = "https://dl.fedoraproject.org/pub/fedora/linux/releases/28/Everything/x86_64/os/Packages/g/glibc-2.27-8.fc28.x86_64.rpm",
   sha256 = "573ceb6ad74b919b06bddd7684a29ef75bc9f3741e067fac1414e05c0087d0b6"
)
```

and it can then be referenced like this:

```
container_image(
    name = "glibc_only_fedora",
    rpms = ["@glibc//file"],
)
```